src/frontend/Player/createVideojsPlayer.spec.tsx

```diff
@@ -128,20 +136,84 @@ describe('createVideoJsPlayer', () => {
     expect(player.options_.fluid).toBe(true);
     expect(player.options_.language).toEqual('fr');
     expect(player.options_.liveui).toBe(false);
-    expect(player.options_.plugins).toEqual({
-      httpSourceSelector: {
-        default: 'auto',
+    expect(player.options_.responsive).toBe(true);
+    expect(player.options_.html5).toEqual({
+      vhs: {
+        overrideNative: true,
       },
     });
+  });
+
+  it('creates videojs player without HLS compat and configures it', () => {
+    mockIsMSESupported.mockReturnValue(false);
+    const { container } = render(
+      wrapInIntlProvider(
+        <VideoPlayer video={mockVideo} playerType={'videojs'} />,
```

Comment:
I know that you will address this point in a separate PR, just to target this line.
`<VideoPlayer video={mockVideo} playerType="videojs" />`